src/content/ja/updates/2016/04/credential-management-api.md

```diff
@@ -353,9 +353,12 @@ mediation モードを有効にすることができます。
 それをあなたの Chrome 拡張機能にフックすることはできるかもしれません。
 
 ## リソース
+より深く Credential Management API を学ぶには、[Integration Guide](https://developers.google.com/web/fundamentals/security/credential-management/) をご覧下さい。
```

Comment:
For internal links, please use absolute paths only, do not include the domain. For example `/web/fundamentals/security/credential-management/`